app/controllers/concerns/filterable.rb

```diff
@@ -25,7 +25,7 @@ def filter_query
   end
 
   def valid_filters?
-    filter_query.map { |a| a.gsub('filter', '') }.all? { |s| s =~ /\A\[\[.+\]\[.+\]\]=.+\z/ }
+    filter_query.map { |a| a.gsub('filter', '') }.all? { |s| s =~ /\A\[[^\]]+\]\[[^\]]+\]=.+\z/ }
```

Comment:
Here is a Rubular playground to help understand this change: https://rubular.com/r/ia4j0fSIiGTc5f